docs/src/tutorials/conic/simple_examples.jl

```diff
@@ -158,7 +158,7 @@ function example_k_means_clustering()
     end
     model = Model(Clarabel.Optimizer)
     set_silent(model)
-    @variable(model, Z[1:m, 1:m] >= 0, PSD)
+    @variable(model, Z[1:m, 1:m] >= 0, set = PSDCone())
```

Comment:
This one is a reason not to stop using `PSD`. It's clunky, and I can see people getting tripped up over it.

Reply:
I like it with the "set", the PSD one does not look better to me

Reply:
Would you think its clearer if we replace ">=0" by "lower_bound=0".
I would not mind.

Reply:
No I want to keep this as it was... We have nice syntax. There's no need to use `lower_bound=0` like we're a Python library.

Reply:
Was the main reason to stop using PSD because `VariableInSet` is confusing?

Reply:
I dont agree that
x >= 0, PSD
Is better than
x >= 0, set = ... 

We should decide if its legacy or not.
For me it felt like it was.
For @blegat the same.

We can keep it, but if we do, we should have PSDRef.

Reply:
> We can keep it, but if we do, we should have PSDRef.

We have VariableInSetRef though?

What about https://github.com/jump-dev/JuMP.jl/pull/3960

Reply:
I'm missing `x >= 0, PSD)`. I vote we leave as-is and wait for more people to ask how to get the dual of a variable constraint.